napari/tests/test_viewer_layer_parity.py

```diff
@@ -29,6 +29,10 @@
 def test_docstring(layer):
     name = layer.__name__
 
+    # Stop testing Image
```

Comment:
This is a little too drastic. Can we check that all arguments in Image are in add_image and view_image?

Reply:
I have done this for add_image.

We don't actually have any doc string tests for any `view_*` methods, but should add them in a new PR